### PR TITLE
North West 4- Bimbola Lasisi- Html/Css -Week1

### PR DESCRIPTION
**Pull request by blasisi**
Created html and  css blog website Include the following element : Header, articles section , Article and Footer sections using semantic html tags and first-child pseudo-class.

**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

-----
[View rendered .github/pull_request_template.md](https://github.com/blasisi/HTML-CSS-Coursework-Week1/blob/master/.github/pull_request_template.md)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,22 +1,95 @@
...
+            alt=" JavaScript image">
+          <p>
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+            scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into
+            electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of
+            Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like
+            Aldus PageMaker including versions of Lorem Ipsum.
+          </p>
+      </article>
```
Comment (nax2uk) [inline index.html:47]: Well done you have added a title using the `<h2>` tag. 👍🏽
Please see below for improvements:
1. You are missing a closing tag `</header>` in between the `<articles></articles>` block. Actually, you have instead put the closing `</header>` tag outside the `<articles></articles>` block
2. The task also requires you to include a `<summary></summary>` block within the `<articles></articles>` block. Please can you add that to your code.

File: index.html [hunk 2]
```diff
@@ -1,22 +1,95 @@
...
+          <img src="https://www.tutorialrepublic.com/lib/images/javascript-illustration.png" alt=" JavaScript image">
+          <p>
+            It is a long established fact that a reader will be distracted by the readable content of a page when
+            looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of
+            letters, as opposed to using 'Content here, content here', making it look like readable English. Many
+            desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a
+            search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved
+            over the years, sometimes by accident, sometimes on purpose (injected humour and the like).
+          </p>
+      </article>
```
Comment (nax2uk) [inline index.html:64]: Well done you have added a title using the `<h2>` tag. 👍🏽
Please see below for improvements:
1. You are missing a closing tag `</header>` in between the `<articles></articles>` block. Actually, you have instead put the closing `</header>` tag outside the `<articles></articles>` block
2. The task also requires you to include a `<summary></summary>` block within the `<articles></articles>` block. Please can you add that to your code.

File: index.html [hunk 3]
```diff
@@ -1,22 +1,95 @@
...
+            alt=" An introduction to JavaScript image">
+          <p> Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical
+            Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at
+            Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem
+            Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable
+            source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et Malorum" (The Extremes
+            of Good and Evil) by Cicero, written in 45 BC. This book is a treatise on the theory of ethics, very popular
+            during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in
+            section 1.10.32.</p>
+      </article>
```
Comment (nax2uk) [inline index.html:82]: Please see previous two comments above regarding the closing `</header>` tag and `<summary>` tag and improve on this code. Thanks.

**Review by nax2uk — COMMENTED** [review]
Well done on completing this assignment! 🎉
Please see my separate comments and it would be good if you could improve on it.
Let me know if you have any questions. 🙂

File: index.html (hunk 1 above)
Comment (blasisi) [reply index.html:47]: Thank you , I will work on it Asap

**Pull request closed without merging**